### ISSUE TITLE
LoadError for 'braintreehttp' file

### ISSUE THREAD

**Issue by jlcarruda** [paypal/Checkout-Ruby-SDK#9]
Hello Braintree ppl.
I'm having trouble trying to integrate the gem in a project of mine. I'm currently using Rails 5.1.6 and Ruby 2.2.2.

The behavior happens when I try to run the server after I execute ```bundle install ```:

```
/Users/lucas/.rvm/gems/ruby-2.2.2/gems/activesupport-5.1.6/lib/active_support/dependencies.rb:292:in `require': cannot load such file -- braintreehttp (LoadError)
  from /Users/lucas/.rvm/gems/ruby-2.2.2/gems/activesupport-5.1.6/lib/active_support/dependencies.rb:292:in `block in require'
  from /Users/lucas/.rvm/gems/ruby-2.2.2/gems/activesupport-5.1.6/lib/active_support/dependencies.rb:258:in `load_dependency'
  from /Users/lucas/.rvm/gems/ruby-2.2.2/gems/activesupport-5.1.6/lib/active_support/dependencies.rb:292:in `require'
  from /Users/lucas/.rvm/gems/ruby-2.2.2/gems/paypal-checkout-sdk-1.0.1/lib/core/paypal_environment.rb:1:in `<top (required)>'
  from /Users/lucas/.rvm/gems/ruby-2.2.2/gems/paypal-checkout-sdk-1.0.1/lib/lib.rb:2:in `require_relative'
  from /Users/lucas/.rvm/gems/ruby-2.2.2/gems/paypal-checkout-sdk-1.0.1/lib/lib.rb:2:in `<top (required)>'
  from /Users/lucas/.rvm/gems/ruby-2.2.2/gems/paypal-checkout-sdk-1.0.1/lib/paypal-checkout-sdk.rb:1:in `require_relative'
  from /Users/lucas/.rvm/gems/ruby-2.2.2/gems/paypal-checkout-sdk-1.0.1/lib/paypal-checkout-sdk.rb:1:in `<top (required)>'
  from /Users/lucas/.rvm/gems/ruby-2.2.2@global/gems/bundler-1.16.6/lib/bundler/runtime.rb:81:in `require'
  from /Users/lucas/.rvm/gems/ruby-2.2.2@global/gems/bundler-1.16.6/lib/bundler/runtime.rb:81:in `block (2 levels) in require'
  from /Users/lucas/.rvm/gems/ruby-2.2.2@global/gems/bundler-1.16.6/lib/bundler/runtime.rb:76:in `each'
  from /Users/lucas/.rvm/gems/ruby-2.2.2@global/gems/bundler-1.16.6/lib/bundler/runtime.rb:76:in `block in require'
  from /Users/lucas/.rvm/gems/ruby-2.2.2@global/gems/bundler-1.16.6/lib/bundler/runtime.rb:65:in `each'
  from /Users/lucas/.rvm/gems/ruby-2.2.2@global/gems/bundler-1.16.6/lib/bundler/runtime.rb:65:in `require'
  from /Users/lucas/.rvm/gems/ruby-2.2.2@global/gems/bundler-1.16.6/lib/bundler.rb:114:in `require'
  from /Users/lucas/Documents/dev/project/config/application.rb:7:in `<top (required)>'
  from /Users/lucas/.rvm/gems/ruby-2.2.2/gems/railties-5.1.6/lib/rails/commands/server/server_command.rb:133:in `require'
  from /Users/lucas/.rvm/gems/ruby-2.2.2/gems/railties-5.1.6/lib/rails/commands/server/server_command.rb:133:in `block in perform'
  from /Users/lucas/.rvm/gems/ruby-2.2.2/gems/railties-5.1.6/lib/rails/commands/server/server_command.rb:130:in `tap'
  from /Users/lucas/.rvm/gems/ruby-2.2.2/gems/railties-5.1.6/lib/rails/commands/server/server_command.rb:130:in `perform'
  from /Users/lucas/.rvm/gems/ruby-2.2.2/gems/thor-0.20.0/lib/thor/command.rb:27:in `run'
  from /Users/lucas/.rvm/gems/ruby-2.2.2/gems/thor-0.20.0/lib/thor/invocation.rb:126:in `invoke_command'
  from /Users/lucas/.rvm/gems/ruby-2.2.2/gems/thor-0.20.0/lib/thor.rb:387:in `dispatch'
  from /Users/lucas/.rvm/gems/ruby-2.2.2/gems/railties-5.1.6/lib/rails/command/base.rb:63:in `perform'
  from /Users/lucas/.rvm/gems/ruby-2.2.2/gems/railties-5.1.6/lib/rails/command.rb:44:in `invoke'
  from /Users/lucas/.rvm/gems/ruby-2.2.2/gems/railties-5.1.6/lib/rails/commands.rb:16:in `<top (required)>'
  from /Users/lucas/Documents/dev/project/bin/rails:9:in `require'
  from /Users/lucas/Documents/dev/project/bin/rails:9:in `<top (required)>'
  from /Users/lucas/.rvm/gems/ruby-2.2.2/gems/spring-2.0.2/lib/spring/client/rails.rb:28:in `load'
  from /Users/lucas/.rvm/gems/ruby-2.2.2/gems/spring-2.0.2/lib/spring/client/rails.rb:28:in `call'
  from /Users/lucas/.rvm/gems/ruby-2.2.2/gems/spring-2.0.2/lib/spring/client/command.rb:7:in `call'
  from /Users/lucas/.rvm/gems/ruby-2.2.2/gems/spring-2.0.2/lib/spring/client.rb:30:in `run'
  from /Users/lucas/.rvm/gems/ruby-2.2.2/gems/spring-2.0.2/bin/spring:49:in `<top (required)>'
  from /Users/lucas/.rvm/gems/ruby-2.2.2/gems/spring-2.0.2/lib/spring/binstub.rb:31:in `load'
  from /Users/lucas/.rvm/gems/ruby-2.2.2/gems/spring-2.0.2/lib/spring/binstub.rb:31:in `<top (required)>'
  from /Users/lucas/.rvm/rubies/ruby-2.2.2/lib/ruby/site_ruby/2.2.0/rubygems/core_ext/kernel_require.rb:70:in `require'
  from /Users/lucas/.rvm/rubies/ruby-2.2.2/lib/ruby/site_ruby/2.2.0/rubygems/core_ext/kernel_require.rb:70:in `require'
  from /Users/lucas/Documents/dev/project/bin/spring:15:in `<top (required)>'
  from bin/rails:3:in `load'
  from bin/rails:3:in `<main>'
```

I know the error happens when the gem is not found and can't be loaded, but the ```paypal-checkout-sdk``` gem is supposed to have this gem as dependency, and therefore the bundler needed to install it as well.

**Comment by jlcarruda** [comment]
As resolution, I needed to add the ```braintreehttp``` gem into my Gemfile as well.

**Comment by rajarampadmanathan** [comment]
HI @jlcarruda - thanks for reporting the issue.. Will look into it and get back to you.

**Comment by KohheePeace** [comment]
@rajarampadmanathan I had the same issue. Rails (5.2.1) and ruby '2.5.1'.

**Comment by rajarampadmanathan** [comment]
Hello, Thanks for reporting this. @jwilger already did the changes to fix this issue. Will release the new version with the changes.

**Comment by AlexWayfer** [comment]
> I Will release the new version with the changes.

How soon?

**Comment by rajarampadmanathan** [comment]
Sorry for the delay. Will release an new version with these changes before end of this week.

**Comment by rajarampadmanathan** [comment]
Released `1.0.2` version. Please validate and let me know in case of any issues.

**Comment by jlcarruda** [comment]
@rajarampadmanathan sorry for the delay.
Tested with the new version. The lib was installed correctly!
